go.mod

```diff
@@ -25,9 +25,9 @@ require (
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.85.0
 	github.com/prometheus/alertmanager v0.28.1
 	github.com/prometheus/client_golang v1.23.0
-	github.com/prometheus/common v0.65.0
+	github.com/prometheus/common v0.65.1-0.20250801071412-c79a891c6c28
 	github.com/prometheus/exporter-toolkit v0.14.0
-	github.com/prometheus/prometheus v0.305.0
+	github.com/prometheus/prometheus v0.305.1-0.20250818080900-0a40df33fb4e
```

Comment:
For utilising the change https://github.com/prometheus/prometheus/commit/0a40df33fb4e23695a5eff5f387aa7651c5ec8cf#diff-b8c18b1e4cc9c9beec1598127742d0a82c9ffb2805996e1887aaa1021ed90587

Reply:
unfortunately this change is not included in prometheus 3.6 so had to get commit hash

Reply:
It's a bit unfortunate but can we add a comment at least?